src/stream-analytics/azext_stream_analytics/_help.py

```diff
@@ -0,0 +1,370 @@
+# coding=utf-8
+# --------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+# --------------------------------------------------------------------------------------------
+
+# pylint: disable=too-many-lines
+# pylint: disable=line-too-long
+from knack.help_files import helps  # pylint: disable=unused-import
+
+
+helps['stream-analytics job'] = """
+    type: group
+    short-summary: Commands to manage stream-analytics streaming job.
+"""
+
+helps['stream-analytics job create'] = """
+    type: command
+    short-summary: Creates a streaming job or replaces an already existing streaming job.
+    examples:
+      - name: Create a streaming job
+        text: |-
+               az stream-analytics job create --resource-group "sjrg6936" --name "sj59" \\
```

Comment:
for resource names in sample, let's consistent with `myResource`, such as myResourceGrouop etc.

Reply:
thanks, updated.